designs/virtualization/4.6/vnc-in-new-window/vnc-in-new-window.md

```diff
@@ -0,0 +1,47 @@
+---
+parent: Virtualization
+version: 4.6
+---
+# VNC in a new window / tab
+
+## In the Administrator console
+
+The Console tab allows the user to connect an in-browser or desktop-based Graphical and/or Serial console to the virtual machine.
+We’ll add a new button ‘Open Console’.
+Users will be able to change the console type using the dropdown selector. The Graphical VNC console is selected by default and will open in a separate window or tab of the console view when clicking the ‘Open console’ button. The default will be set to open the console in a new window. Users will be able to drag the tab to be a new window.
+The Expand action makes the console viewport fill the current window’s full width and height.
+
+![open from the new button](img/Open-console-1c.png)
+
+Another way to open the VNC Console can be done via the kebab dropdown menu.
+
+![open from the kebab menu](img/Open-console2.png)
+
+## In the Developer console
```

Comment:
I've reached out to @beaumorley @serenamarie125 about how we want to deal with the dev side of this. We may want to push it out to another release or put the design under the dev console designs.

Reply:
ok we are ok I've confirmed that this is already pulled into the dev perspective. 
I did noticed though that the icon for a new window/tab didn't get added so we could file a bug for that.